Let's clean up the Makefile ... we want to:
 - make sure COPTS is clean for clobbering with user $CFLAGS
 - make sure LOPTS is clean for clobbering with user $LDFLAGS
 - remove all references to config.h since we generate it for the 
   user and we really do NOT want the interactive configure running
 - make sure libdir depends on version.h to fix parallel builds

--- Makefile.orig	2005-01-10 13:11:00.369702792 -0500
+++ Makefile	2005-01-10 13:15:36.542718112 -0500
@@ -88,7 +88,7 @@
 
 # Compiler and Linker Options
 # You may need to uncomment and edit these if you are using libc5 and IPv6.
-COPTS = -D_GNU_SOURCE -O2 -Wall -g # -I/usr/inet6/include
+COPTS = 
 ifeq ($(origin LOPTS), undefined)
 LOPTS = 
 endif
@@ -113,7 +113,7 @@
 
 NET_LIB = $(NET_LIB_PATH)/lib$(NET_LIB_NAME).a
 
-CFLAGS	= $(COPTS) -I. -idirafter ./include/ -I$(NET_LIB_PATH)
+CFLAGS	= $(COPTS) -I. -idirafter ./include/ -I$(NET_LIB_PATH) -D_GNU_SOURCE -Wall
 LDFLAGS	= $(LOPTS) -L$(NET_LIB_PATH)
 
 SUBDIRS	= man/ $(NET_LIB_PATH)/
@@ -127,12 +127,12 @@
 
 MDEFINES = COPTS='$(COPTS)' LOPTS='$(LOPTS)' TOPDIR='$(TOPDIR)'
 
-%.o:		%.c config.h version.h intl.h net-features.h $<
+%.o:		%.c version.h intl.h net-features.h $<
 		$(CC) $(CFLAGS) -c $<
 
-all:		config.h version.h subdirs $(PROGS)
+all:		version.h subdirs $(PROGS)
 
-config: 	cleanconfig config.h
+config: 	cleanconfig
 
 install:	all savebin installbin installdata
 
@@ -159,14 +159,6 @@
 		@tar -cvz -f ../net-tools-$(RELEASE).tar.gz -C .. net-tools
 
 
-config.h: 	config.in Makefile 
-		@echo "Configuring the Linux net-tools (NET-3 Base Utilities)..." ; echo
-		@if [ config.status -nt config.in ]; \
-			then ./configure.sh config.status; \
-		   else ./configure.sh config.in; \
-		 fi
-
-
 version.h:	Makefile
 		@echo "#define RELEASE \"net-tools $(RELEASE)\"" >version.h
 
@@ -175,7 +167,7 @@
 
 i18n.h:		i18ndir
 
-libdir:
+libdir: version.h
 		@$(MAKE) -C $(NET_LIB_PATH) $(MDEFINES)
 
 i18ndir:
